java-rest-service/accelerator.yaml

```diff
@@ -223,12 +219,19 @@ engine:
         chain:
           - type: RewritePath
             rewriteTo: "'config/workload.yaml'"
+    # End OpenAPI Endpoint Auto-registration
+
+    # Workload Resource name
+    - merge:
+      - exclude: [ "config/workload.yaml", "config/test-pipeline.yaml" ]
       - include: [ "config/workload.yaml", "config/test-pipeline.yaml" ]
```

Comment:
why exclude and include the same set of files?

Reply:
maybe add `     - include: [ "**" ]` before the exclude to make it more obvious

Reply:
and that is also the pattern used in "# OpenAPI Endpoint Auto-registration"

Reply:
Merge require us to exclude the files we would like to change in order to keep all files.